### ISSUE TITLE
Color thresholds for values below 0 (plus optionally allowing >/< syntax)

### ISSUE THREAD

**Issue by bcutter** [maxwroc/battery-state-card#482]
**Is your feature request related to a problem? Please describe.**
With HA Companion app 2022.8 for Android the battery power sensor has been introduced. This gives the current power usage, e. g.:
![grafik](https://user-images.githubusercontent.com/13799156/187081913-0699f406-97a1-4e94-a64e-f0b91fb14a0f.png)
Currently a color threshold can not be set for negative values (<0). Syntactically it's also not possible to work with "great than" and "lower than".

**Describe the solution you'd like**
At least allow negative values to work with color_thresholds. >/< would be an optional nice thing. So either:
```
    color_thresholds:
      - value: <0
        color: red
      - value: 0
        color: yellow
      - value: >0
        color: green
```
or at least
```
    color_thresholds:
      - value: -0.01
        color: red
      - value: 0
        color: yellow
      - value: 0.01
        color: green
```
Currently for negative values the color_threshold is not applied, even if value is set as `0`.

**Additional context**
- [Release notes HA Companion App for Android](https://github.com/home-assistant/android/releases/tag/2022.8.0)
- [Details on new battery power sensor](https://github.com/home-assistant/android/pull/2799)
- [Similar issue related to color threshold setting](https://github.com/maxwroc/battery-state-card/issues/460)

**Comment by maxwroc** [comment]
What if you use multiplier? I think it should work but of course it will modify the displayed value.
```yaml
multiplier: -1
```

Does it solve your problem?

**Comment by Scoff123** [comment]
Catering for -ve values would be very helpful as I would like to use the color_gradient for my BLE sensor signal strength, which measures in -dBm vales, however the color_threshold does not work for these values.

**Comment by maxwroc** [comment]
In v3 you have the option to convert value to absolute and convert it to desired percentage via KString/thresholds function.

**Comment by bcutter** [comment]
> In v3 you have the option to convert value to absolute and convert it to desired percentage via KString/thresholds function.

@maxwroc Unfortunately I neither see a "lowerthan" option (only `"{state|greaterthan(10,100)}"`) nor a single example visualizing negative values.

So even the technical possibility might exist now - it's way to complicated to use without a proper example in the docs.

**Comment by maxwroc** [comment]
TBH It would be quite hard/impossible to put examples for all of the cases and combinations in the doc.

> Unfortunately I neither see a "lowerthan" option

The name of it is "lessthan" ([Less-than_sign](https://en.wikipedia.org/wiki/Less-than_sign))

>  nor a single example visualizing negative values

This is not true. Here is the example with negative values: https://github.com/maxwroc/battery-state-card#rssi-sensors-signal-strength

The way to achieve it is the following:
* `"{state|abs()|greaterthan(10,100)}"`
  This converts the state value to the absolute and later changes the value to 100 if the state value is greater than 10.
* `"{state|abs()|lessthan(10,0)}"`
  Similar as above but changes value to 0 if state is lower than 10
* You can combine both as well
  `"{state|abs()|lessthan(10,0)}|greaterthan(10,100)}"`
  Documentation states this as well:
  ![image](https://github.com/maxwroc/battery-state-card/assets/8268674/24bc6cdc-e850-48eb-b5c0-b25b051704ad)

To use the above and change the shown entity state value you need to use the "value_override" in your config (please take a look at the example which I have linked above)

I will think about adding more examples to the doc, but as I have said, card is getting more and more features and it would be quite hard to cover all of their combinations in the doc.

If the above solved your problem feel free to contribute and add an example in the doc. If you're still confused and don't know how to achieve the expected result please let me know.

BTW Thanks for such detailed description of the issue with HA doc/pr links etc!

**Comment by bcutter** [comment]
I played a bit with the `value_override`. Hell now, what a pain. The problem here is, that
- the override changes the values itself which is completely irrelevant here and creates other visualization issues, it's **just about the color_thresholds**
- so I can not find a override template which keeps the current values. In detail:

Few examples:
1) normal with value_override:
![grafik](https://github.com/maxwroc/battery-state-card/assets/13799156/388bd11d-1c9f-4a15-a7bf-5ba3964b4ba0)

2) with `value_override: "{state|abs()|lessthan(0,-10)}"`
![grafik](https://github.com/maxwroc/battery-state-card/assets/13799156/a1de16a7-0e1a-477e-8910-c373593a116c)
--> why the hell are the (actually negative, below 0) values now shown as positive ones?

The main issue is that the color formatting is dependent on the value. And the value_override stuff is mighty, but obviously not working well (or I'm using it wrong). What a pity having such a powerful tool and not being able to do such simple things.

**In other words: how to show entities in green, yellow and red depending on their range from e. g. this to that?**
![grafik](https://github.com/maxwroc/battery-state-card/assets/13799156/6ca15d08-fa96-4174-bdd5-77dea978dae7)

**Comment by maxwroc** [comment]
There was an attempt to add such support but there were some issues and it had to be reverted https://github.com/maxwroc/battery-state-card/pull/599

It is possible to do it but it just requires little bit more thinking. 

For example I don't want to enable setting colors for any value ranges and then later hear people complaining that the gradient option doesn't work with them etc.

@amsterjanis maybe you would be interested to tackle this problem again?
